src/operator/tensor/cast_storage.cc

```diff
@@ -32,6 +32,39 @@ namespace op {
 DMLC_REGISTER_PARAMETER(CastStorageParam);
 NNVM_REGISTER_OP(cast_storage)
 .describe(R"code(Casts tensor storage type to the new type.
+
+When an NDArray with default storage type is casted to csr or row_sparse storage,
```

Comment:
`casted` was the past tense of `cast` in old English. It's `cast` nowadays.
https://english.stackexchange.com/questions/94565/can-casted-be-the-past-tense-of-cast